internal/datacoord/meta.go

```diff
@@ -182,6 +185,14 @@ func (m *meta) reloadFromKV() error {
 		m.clusteringCompactions[fmt.Sprintf("%d-%d", info.CollectionID, info.TriggerID)] = info
 	}
 
+	partitionStatsInfos, err := m.catalog.ListPartitionStatsInfos(m.ctx)
+	if err != nil {
+		return err
+	}
+	for _, info := range partitionStatsInfos {
+		m.partitionStatsInfos[fmt.Sprintf("%d/%d/%s/%d", info.CollectionID, info.PartitionID, info.VChannel, info.Version)] = info
```

Comment:
why use info.Version and the part of key?

Reply:
PartitionStats has version. One partition may have multi partitionStats,  clustering_compaction_manager control the  version update and gc logic.